sorc/global_cycle.fd/cycle.f90

```diff
@@ -304,8 +308,9 @@ SUBROUTINE SFCDRV(LUGB, IDIM,JDIM,LENSFC,LSOIL,DELTSFC,  &
  USE READ_WRITE_DATA
  use machine
  USE MPI
- USE LAND_INCREMENTS, ONLY: ADD_INCREMENT_SOIL,     &
-                            ADD_INCREMENT_SNOW,     &
+ USE LAND_INCREMENTS, ONLY: ADD_GSI_INCREMENT_SOIL,     &
```

Comment:
It would be much better if we could collapse Add_GSI_INCREMENT_SOIL and ADD_JEDI_INCREMENT_SOIL into a single routine, with the interpolation done at the top if necessary. Then the actual adding is done next.

Reply:
Will do.